scripts/helpers/downloads.js

```diff
@@ -147,6 +147,11 @@ module.exports = (version) => {
     downloads = downloads.filter(ver =>
       ver.title !== 'Linux 32-bit Binary' &&
       ver.title !== 'SmartOS 32-bit Binary')
+  } else if (semver.satisfies(version, '>= 12.0.0')) {
```

Comment:
IIUC, this condition seems unreachable since it would be caught by `semver.satisfies(version, '>= 10.0.0')` in the line 146:
https://github.com/nodejs/nodejs.org/blob/8216eeb1311c5a7056fb17b47c12113cf81b06d8/scripts/helpers/downloads.js#L146

Reply:
You are right, of course. Stupid copy & paste mistake.
I refactored the download helper, PTAL.